### PR TITLE
Docs error fix

### LINKED ISSUE #1833: typo and grammatical error in  discord-readme.md and  CODE_OF_CONDUCT.md
### Description : 
some typo and grammatical errors like a word missing and some punctuation error in docs.

### PR DESCRIPTION
**Pull request by shashank-001**
closes #1833 

#### Description of Task to be completed?
Resolved typo and grammatical error in discord-readme.md and CODE_OF_CONDUCT.md file 

#### What is the relevant issue link?
https://github.com/EddieHubCommunity/support/issues/1833#issue-917084666

### CODE REVIEW

**Comment by schmelto** [comment]
Above only my ideas.
Otherwise it's looking good for me 🚀

File: discord-readme.md [hunk 1]
```diff
@@ -47,14 +47,14 @@ GENERAL DISCUSSION:
...
+-> #bot-chat: discuss bots, esp. regarding development of EddieBot. This is also a place to check your profile and ranks in the community using the bot or use the other bot commands.
 DAILY:
 
 -> #healthfitness: share your tips on health and fitness so we can be healthy together.
 
--> #standup: everyday of code, give us your standup updates! What you did yesterday,what you plan to do today and any blockers?
+-> #standup: every day of code, give us your standup updates! What you did yesterday, what you plan to do today, and any blockers?
 
--> #achievements: share your personal achievements with the community
+-> #achievements: share your achievements with the community
```
[the remark this answers is not in the document]
Comment (shashank-001) [reply discord-readme.md:57]: we used *your* which symbolize that it's your personal  but yeah we can keep that  under braces as well

[the remark this answers is not in the document]
Comment (schmelto) [reply discord-readme.md:57]: oh I see your intention behind it. Feel free to leave it as it is without the `personal` think both point of views are correct :)

File: discord-readme.md [hunk 2]
```diff
@@ -27,9 +27,9 @@ The following are the guidelines regarding chat on the discord server:
 
 Instances of abusive, harassing or otherwise, unacceptable behaviour may be reported by contacting the @moderators . All complaints will be reviewed and investigated and will result in a response that is deemed necessary and appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
 
-EddieBotDev (or EddieBot), would be monitoring language used on the discord server, to ensure inclusive and welcoming language in the discord server. EddieBot suggests alternatives and it would be a good practice to edit your message, reflecting the suggestion
+EddieBotDev (or EddieBot), would be monitoring language used on the discord server, to ensure the inclusive and welcoming language in the discord server. EddieBot suggests alternatives and it would be a good practice to edit your message, reflecting the suggestion
```
[the remark this answers is not in the document]
Comment (shashank-001) [reply discord-readme.md:30]: **the** : add because it's determiners and we should start noun phrases with it.

**Review by schmelto — DISMISSED** [review]

**Review by schmelto — APPROVED** [review]

**Comment by schmelto** [comment]
Thanks for you contribution :) see you in some other PRs or in the Discord community 🚀

**Pull request merged**